### PR TITLE
Feat/439716  add police user

### PR DESCRIPTION
**Pull request by whitewaterdesign**

### CODE REVIEW

File: app/routes/admin/police/remove.js [hunk 1]
```diff
@@ -125,7 +125,7 @@ module.exports = [
     options: {
       auth: { scope: [admin] },
       validate: {
-        payload: validatePayloadBuilder(confirmFlowValidFields(addRemoveConstants.inputField, addRemoveConstants.messageLabel))
+        payload: validatePayloadBuilder(confirmFlowValidFields(addRemoveConstants.inputField))
```
Comment (whitewaterdesign) [inline app/routes/admin/police/remove.js:128]: useless argument

File: app/views/admin/success.njk [hunk 2]
```diff
@@ -33,6 +33,27 @@
...
+            {% if item.ul  %}
+                <ul class="govuk-list govuk-list--bullet">
+                    {% for item in item.ul %}
+                        <li>{{item.text}}</li>
+                    {% endfor %}
+                </ul>
+            {% endif %}
+          {% endfor %}
+      {% endif %}
+
```
Comment (whitewaterdesign) [inline app/views/admin/success.njk:55]: so we can reuse success page

File: app/models/admin/courts/builder.js [hunk 3]
```diff
@@ -33,7 +33,49 @@ const CourtRemovedViewModel = (court) => {
   })
 }
 
+const PoliceOffersAddedViewModel = (officers) => {
```
Comment (jbarnsley10) [inline app/models/admin/courts/builder.js:36]: This should probably be in a file for police (not courts) but not going to quibble as we don't have a lot of time this week :)

File: app/schema/portal/common/single-submit.js [hunk 4]
```diff
@@ -1,8 +1,16 @@
 const Joi = require('joi')
 
-const confirmFlowValidFields = (field) => {
+const confirmFlowValidFields = (field, fields = []) => {
+  const moreFields = fields.reduce((base, newField) => {
```
Comment (jbarnsley10) [inline app/schema/portal/common/single-submit.js:4]: If you're trying to allow extra field names to not break the validation, I think you can just add '.unknown()' or something to the schema

File: app/views/admin/success.njk [hunk 5]
```diff
@@ -33,6 +33,27 @@
       <p class="govuk-body">{{ paragraph }}</p>
       {% endfor %}
 
+      {% if model.html %}
```
Comment (jbarnsley10) [inline app/views/admin/success.njk:36]: Do we need to be careful of code injection here? I guess it's admin users, and the WAF should block major issues

File: app/views/admin/users/add/confirm.njk [hunk 6]
```diff
@@ -0,0 +1,56 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "components/statusTag/macro.njk" import statusTag %}
```
Comment (jbarnsley10) [inline app/views/admin/users/add/confirm.njk:3]: A few unnecessary imports

File: app/views/admin/users/add/list.njk [hunk 7]
```diff
@@ -0,0 +1,55 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "components/statusTag/macro.njk" import statusTag %}
```
Comment (jbarnsley10) [inline app/views/admin/users/add/list.njk:3]: unnecessary import

**Review by jbarnsley10 — APPROVED** [review]
Really excellent. A couple of comments but I suggest none require any changes as we're short of time, and they're only minor anyway

**Pull request merged**